Src/library/cURL/interface/curl_easy_externals_i.e

```diff
@@ -115,6 +115,16 @@ feature -- Command
 			setopt_void_star (a_curl_handle, a_opt, a_file.file_pointer)
 		end
 
+	setopt_mime (a_curl_handle: POINTER; a_opt: INTEGER; a_mime: POINTER)
+			-- Declared as curl_easy_setopt().
```

Comment:
Not sure if this is a typo or not about "curl_easy_setopt()"
but it is, then the previous features have the same typo.

Reply:
Not sure, all the features calling the external C function used the same convention, maybe to indicate that was C function, I've updated all the comments.